app/services/work_index_jobs.rb

```diff
@@ -38,10 +38,12 @@ def call
                        status: DoneIndexJob::STATUS_SUCCESSFUL,
                        es_stats: es_stats,
                        time_took: time_took)
-    rescue InvalidIndexingStrategy
+    rescue InvalidIndexingStrategy => ex
+      Raven.capture_exception(ex)
```

Comment:
so here, with this, we'll end up with 2 Sentry exceptions:  one from here, and then one later, when the done job is processed w/ a INVALID_INDEXING_STRATEGY.   Is this ok?

Reply:
Yes